post-microsservice/src/main/kotlin/com/example/postmicrosservice/model/Post.kt

```diff
@@ -5,10 +5,9 @@ import jakarta.persistence.GeneratedValue
 import jakarta.persistence.Id
 
 @Entity
-class Post (
+data class Post (
```

Comment:
Why did you changed that ?

Reply:
Because it's a class for a DB entity, so we don't need any method related to it. So, in this case, we use `dataClass`